jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,471 @@
+    "PL_API_KEY = os.environ.get('PL_API_KEY')\n",
+    "GCP_CREDENTIALS = os.environ.get('GCP_CREDENTIALS')\n",
+    "auth = planet.Auth.from_key(PL_API_KEY)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create an order packet\n",
+    "---\n",
+    "Package up the details of your order in a [JSON object](https://developers.planet.com/apis/orders/basemaps/#example-order-query-json-block) and make a POST request, passing in the Orders URL, your JSON, your API key, and the content-type header. We are [delivering](https://developers.planet.com/apis/orders/delivery/) this order to a Google Cloud Storage bucket. In this request, we are also using the tools [merge](https://developers.planet.com/apis/orders/tools/#merge) and [clip](https://developers.planet.com/apis/orders/tools/#clip). "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "order_params = {\n",
+    "    \"name\": \"basemap order with geometry\",\n",
+    "    \"source_type\": \"basemaps\",\n",
+    "    \"order_type\": \"partial\",\n",
+    "    \"products\": [\n",
+    "        {\n",
+    "            \"mosaic_name\": \"global_monthly_2022_01_mosaic\",\n",
+    "            \"geometry\": {\n",
+    "                \"type\": \"Polygon\",\n",
+    "                \"coordinates\": [\n",
+    "                   [\n",
+    "                       [4.607406, 52.353994],\n",
+    "                       [4.680005, 52.353994],\n",
+    "                       [4.680005, 52.395523],\n",
+    "                       [4.607406, 52.395523],\n",
+    "                       [4.607406, 52.353994]\n",
+    "                   ]\n",
+    "                ]\n",
+    "            }\n",
+    "        }\n",
+    "    ],\n",
+    "    \"tools\": [\n",
+    "        {\"merge\": {}},\n",
+    "        {\"clip\": {}}\n",
+    "    ],\n",
+    "    \"delivery\": {\n",
+    "        \"google_cloud_storage\": {\n",
+    "            \"bucket\": \"devrel-notebooks\",\n",
+    "            \"credentials\": GCP_CREDENTIALS,\n",
+    "            \"path_prefix\": \"basemaps-to-cloud/\"\n",
+    "        }\n",
+    "    }\n",
+    "}"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create a session with SDK and poll for success\n",
+    "\n",
+    "Here, we are creating an order using the SDK V2. Then we are waiting for the order to be delievered to our GCP bucket."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "async def create_and_deliver_order(order_params, client):\n",
+    "    '''Create an order and wait for it to delivered\n",
+    "\n",
+    "    Parameters:\n",
+    "        order_params: An order request\n",
+    "        client: An Order client object\n",
+    "    Return (str): state of the order\n",
```

Comment:
I've given this some thought and I'd recommend removing "Return (str)..." and in the top of the docstring mention that the function prints the state of the order.